content/docs/reference/routes/to.mdx

```diff
@@ -81,6 +81,20 @@ A load balancing weight may be associated with a particular upstream by appendin
   to: ['http://a,10', 'http://b,20']
 ```
 
+### HTTP/2 cleartext
+
+When Pomerium connects to an `https` upstream, it will negotiate either HTTP/1.1 or HTTP/2 using [ALPN](https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation) (as part of the TLS handshake).
+
+To configure Pomerium to make requests to an upstream service using HTTP/2 _without_ TLS (i.e. in cleartext), use the special `h2c://` scheme.
```

Comment:
Small nit: I try to avoid latinate expressions entirely, such as "i.e." (this is covered in a lot of style guides, for example [Google's developer docs style guide](https://developers.google.com/style/abbreviations#dont-use)). Let's update to:
```suggestion
To configure Pomerium to make requests to an upstream service using HTTP/2 _without_ TLS (that is, in cleartext), use the special `h2c://` scheme.
```

Reply:
Sure, sounds good.